src/styles/main.scss

```diff
@@ -1,3 +1,263 @@
-body {
-  margin: 0;
+@import "../styles/utils/variables";
+@import "../styles/utils/mixin";
+@import "../styles/stars-block/stars";
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__nav {
+    margin-right: 50px;
+  }
+}
+
+.logo {
+  margin: 10px 0 10px 50px;
+}
+
+.nav {
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    height: 60px;
+    position: relative;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    color: $main-color-text;
+    text-transform: uppercase;
+    line-height: 60px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: color 0.3s;
+  }
+
+  &__link:hover {
+    color: $main-color;
+  }
+}
+
+.is-active {
+  color: $main-color;
+}
+
+.is-active::after {
+  content: "";
+  text-transform: uppercase;
+  position: absolute;
+  border-bottom: 4px solid $main-color;
+  border-radius: 8px;
+  width: 100%;
+  left: 0;
+  bottom: 0;
+}
+
+.hamburger {
+  display: none;
+}
+
+.bar {
+  display: block;
+  width: 30px;
+  height: 2px;
+  margin: 6px auto;
+  transition: all 0.3s ease-in-out;
+  background-color: $main-color;
+}
+
+@include on-tablet-menu {
+  .body.active {
+    position: fixed;
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+  }
+
+  .header__logo-link {
```

Comment:
that's nice you know about media query :+1: 

styles related to header should be inside `.header` selector - even media queries

Reply:
![Screenshot_8](https://github.com/mate-academy/layout_catalog/assets/149514552/c8acf60a-3960-45c1-8ab1-e0112a832a58)